### DISCUSSION TITLE
Float above all others floating windows

### DISCUSSION THREAD

**Discussion by omnigenous** [Schweber/hdrop#11] [Q&A]
Because kitty inside hdrop is pretty much a first thing that opens on startup it's always under other floating windows. I have entire workspace filling with floating windows, so I would have to aim inside a window gap while holding super key and clicking on kitty after activating hdrop to make it float above all.

Is there a way to make kitty inside hdrop float above all other floating windows including outside hdrop? I tried pinning (`hyprctl dispatch pin class:hdrop`) but that makes kitty visible even when hdrop is hidden.

**Comment by Schweber** [comment]
Sorry for the late response: i don’t know a way to do this out of the top of my head. Maybe you can delay the launch of the kitty window e.g. with "sleep 1 && hdrop kitty" so that it is opened after the other windows and is on top.

Pinning is an interesting idea. If you like, you can edit the hdrop script and add an option to pin a window when it is supposed to be shown and unpin it when it’s supposed to be hidden. I’d be happy to merge this as a PR.

I’m not using hyprland anymore so i likely won’t work on this myself.
